Neos.Media.Browser/Classes/Controller/AssetController.php

```diff
@@ -468,6 +475,32 @@ public function variantsAction(string $assetSourceIdentifier, string $assetProxy
         }
     }
 
+    /**
+     * (Re-)create all variants for the given image
```

Comment:
You're right, that was misleading. Is this better:
```suggestion
     * Create missing variants for the given image
```